R/pairwise.R

```diff
@@ -123,8 +123,13 @@ do_dist.cols <- function(df, ..., label=NULL, fill=0, fun.aggregate=mean, distin
   calc_dist_each <- function(df){
     mat <- df %>%  dplyr::select_(.dots=select_dots) %>%  as.matrix()
 
+    # sort the column name so that the output of pair.name will be sorted
```

Comment:
I think this comment is a little too terse. It took some time for me to understand what it meant.
- You could say "pair.name.1 and pair.name.2" instead of saying "pair.name".
- Does it make sense to explain why it is important or useful that pair.name.1 and pair.name.2 is sorted? If so, please do.